_posts/2025-04-29-juan-luis-opensouce-journey.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title:  "A Journey Through Open Source: From Hacker Culture to the Age of AI"
+description: "A Journey Through Open Source: From Hacker Culture to the Age of AI"
+date:   2025-04-29 10:00:35 +0300
+image:  '/images/blogs/blog-placeholder.png'
+tags:   [opensource, community]
+---
+
+Author: [Reshama Shaikh](https://reshamas.github.io)  
+
```

Comment:
@SangamSwadiK 
You can post as follows (and remove the Author part):
Summary by: Sangam Swadi K